setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = git-machete
+summary = Tool to rebase local branches with theirs parent branches
+description-file = README.md
+license = MIT License
+classifiers =
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 3
+    Programming Language :: Python
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+keywords = git
+author = VirtusLab
+author-email = info@virtuslab.com
+home-page = https://github.com/VirtusLab/git-machete/
+project-url = https://github.com/VirtusLab/
+
+[files]
+packages =
+    git_machete
+
+[entry_points]
+console_scripts =
+    git-machete = git_machete.cmd:main
+
+[wheel]
```

Comment:
I don't have much knowledge about python packaging... but are we gonna use wheels, eggs, or both? Coz this `[wheel]` entry suggests a wheel, and setup.py implies an egg, IIUC